pkg/kv/dynamodb/store.go

```diff
@@ -77,14 +100,17 @@ func (d *Driver) Open(ctx context.Context, dsn string) (kv.Store, error) {
 	}, nil
 }
 
-type Params struct {
-	TableName string
-}
-
-func (s *Store) getPartitionKey(_ /*itemKey*/ []byte) string {
-	// At some point the partition key will probably depend on the key.
-	// Meanwhile, it is simply the table name (i.e. everything is under the same partition)
-	return s.params.TableName
+func parseDsn(dsn string) (*Params, error) {
+	params := &Params{
+		TableName:          DefaultTableName,
+		ReadCapacityUnits:  DefaultReadCapacityUnits,
+		WriteCapacityUnits: DefaultWriteCapacityUnits,
+	}
+	err := json.Unmarshal([]byte(dsn), params)
```

Comment:
Does it keep the values if the parsing succeeded with those keys missing? I learned something new..

Reply:
:star_struck: